src/pigpio.cc

```diff
@@ -687,6 +707,205 @@ NAN_METHOD(gpioNotifyClose) {
   }
 }
 
+/* ------------------------------------------------------------------------ */
+/* Waves                                                                    */
+/* ------------------------------------------------------------------------ */
+
+NAN_METHOD(gpioWaveClear) {
+  int rc = gpioWaveClear();
+  if (rc < 0) {
+    return ThrowPigpioError(rc, "gpioWaveClear");
+  }
+  info.GetReturnValue().Set(rc);
+}
+
+NAN_METHOD(gpioWaveAddNew) {
+  int rc = gpioWaveAddNew();
+  if (rc < 0) {
+    return ThrowPigpioError(rc, "gpioWaveAddNew");
+  }
+  info.GetReturnValue().Set(rc);
+}
+
+NAN_METHOD(gpioWaveAddGeneric) {
+  if (info.Length() < 2 || !info[0]->IsUint32() || info[1]->IsNull()) {
+    return Nan::ThrowError(Nan::ErrnoException(EINVAL, "gpioWaveAddGeneric", ""));
+  }
+
+  unsigned numPulses = info[0]->Uint32Value();
+  v8::Local < v8::Array > array = v8::Local < v8::Array > ::Cast(info[1]);
```

Comment:
Use `v8::Local<v8::Array> array = info[1].As<v8::Array>();` rather than `v8::Local < v8::Array > array = v8::Local < v8::Array > ::Cast(info[1]);`

Reply:
Fixed in cf56d02